api/GamepadButton.json

```diff
@@ -84,9 +98,16 @@
                 "version_removed": "34"
               }
             ],
-            "chrome_android": {
-              "version_added": true
-            },
+            "chrome_android": [
+              {
+                "version_added": "35"
+              },
+              {
+                "prefix": "webkit",
```

Comment:
I don't trust the data being mirrored here. It's not very likely that both the `GamepadButton` interface and the `pressed` property were webkit-prefixed. Most likely the prefix data should be removed from all sub-features here.

Backing out these changes and doing it separately would be best.

Reply:
Opened #13269 to take care of the gamepad data!